README.md

```diff
@@ -48,20 +72,10 @@ OpenFeature::SDK.configure do |config|
       "flag2" => 1
     }
   ))
-  # alternatively, you can bind multiple providers to different domains
-  config.set_provider(OpenFeature::SDK::Provider::NoOpProvider.new, domain: "legacy_flags")
-  # you can set a global evaluation context here
-  config.evaluation_context = OpenFeature::SDK::EvaluationContext.new("host" => "myhost.com")
 end
 
 # Create a client
 client = OpenFeature::SDK.build_client
-# Create a client for a different domain, this will use the provider assigned to that domain
-legacy_flag_client = OpenFeature::SDK.build_client(domain: "legacy_flags")
-# Evaluation context can be set on a client as well
-client_with_context = OpenFeature::SDK.build_client(
-  evaluation_context: OpenFeature::SDK::EvaluationContext.new("controller_name" => "admin")
-)
 
 # fetching boolean value feature flag
 bool_value = client.fetch_boolean_value(flag_key: 'boolean_flag', default_value: false)
```

Comment:
It may be great to add the details function as well in the documentation to be explicit that it is available.